### PR TITLE
change the date to the testopolis seed project to 05.06.2024

### PR DESCRIPTION
**Pull request by lsteinmann**
The choice of date is rather random and manually assigned in the seed-script; in this case it is the last change in the project database. We wanted to change the date to something that in the past so that one could draft a new publication from the testopolis project on Field Hub on the same day as running `mix seed` to get the (local) test project.

### CODE REVIEW

File: publication/test/support/fixtures/seed_project/project_seed.ex [hunk 1]
```diff
@@ -98,7 +99,7 @@ defmodule FieldPublication.Test.ProjectSeed do
 
     {:ok, %FieldPublication.DatabaseSchema.Publication{} = publication} =
       Publications.put(publication, %{
-        "publication_date" => Date.utc_today(),
+        "publication_date" => Date.from_iso8601!("2024-06-05"),
```
Comment (dersmon) [inline publication/test/support/fixtures/seed_project/project_seed.ex:102]: Is this still necessary? 

Actually I am not sure why the old line was here in the first place, the date should have been set in `Publications.create_from_replication_input/1`. With your change, this should be set using the default value instead.

**Review by dersmon — COMMENTED** [review]
:+1: :1st_place_medal:

File: publication/test/support/fixtures/seed_project/project_seed.ex (hunk 1 above)
Comment (lsteinmann) [reply publication/test/support/fixtures/seed_project/project_seed.ex:102]: Well, I tried and it works just fine without this line. I removed it and will merge that.

Comment (lsteinmann) [reply publication/test/support/fixtures/seed_project/project_seed.ex:102]: Was necessary after all, commit #[97d8520](https://github.com/dainst/idai-field/pull/256/commits/97d8520034a0c68a1682265838da344bc2f53a77) is reverted.

**Pull request merged**